src/_rules/list.js

```diff
@@ -0,0 +1,28 @@
+import { entriesToCss } from '@unocss/core';
+
+const listItemStyles = entriesToCss(Object.entries({
+  position: 'relative',
+  'padding-left': 'var(--w-space-3)',
+}));
+const checkmarkStyles = entriesToCss(Object.entries({
+  content: '""',
+  display: 'block',
+  position: 'absolute',
+  width: '16px',
+  height: '28px',
+  left: 0,
+  color: 'var(--w-list-color-icon-checked)',
```

Comment:
```suggestion
  color: 'var(--w-list-color-icon-checked, hotpink)',
```
Should we be adding fallback to all variables?

Reply:
No fallback needed in any of these rules IMO